packages/components/src/components/telekom/telekom-footer-extended-navigation/telekom-footer-extended-navigation-column.tsx

```diff
@@ -41,16 +41,18 @@ export class TelekomFooterExtendedNavigationColumn {
           })}
         >
           <div part="heading-container">
-            <h2 aria-level={this.headingLevel}>
+            <span role="heading" aria-level={this.headingLevel}>
```

Comment:
I would add a part to this, probably called `heading-with-button` and then target it for hiding it on desktop (`display: none`) instead of actually hiding the button inside of it, see https://github.com/telekom/scale/blob/main/packages/components/src/components/telekom/telekom-footer-extended-navigation/telekom-footer-extended-navigation-column.css#L61-L63

Also there's some CSS that got lost [when removing the h2 from the markup, also the for button below…](https://github.com/telekom/scale/blob/main/packages/components/src/components/telekom/telekom-footer-extended-navigation/telekom-footer-extended-navigation-column.css#L47-L55) I would again use a part to target them:

```
[part~='heading-container'] h2 {
  margin-block-start: 0;
  margin-block-end: 0;
}
```